<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_Epsilon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Accuracy</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_World">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>721</width>
      <height>721</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="EpsilonSpinBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>0.010000000000000</double>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.800000000000000</double>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>350</width>
      <height>350</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="interactive">
     <bool>false</bool>
    </property>
    <property name="viewportUpdateMode">
     <enum>QGraphicsView::FullViewportUpdate</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="DrawDistanceR1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DrawDistancesR1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testR1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>RunTestR1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testR2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>440</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>RunTestR2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DrawDistanceR2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DrawDistancesR2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButtton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testContract2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>500</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Contract2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DrawDistanceR3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DrawDistancesR3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testR3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>460</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>RunTestR3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startTimer">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>540</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Timer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="contractState">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>580</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ContractState</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="nbStep">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>120</number>
    </property>
   </widget>
   <widget class="QPushButton" name="drawALL">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>98</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DrawALL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stepR1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>StepR1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1097</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
